File: programming-languages/exercises/cpp/swap_two_variables.cpp
#include <iostream>
using namespace std;
int main(){
    int a=1,b=2,temp=0;
    cout << "before swap " << a << " " << b << endl;
    temp=a;a=b;b=temp;
    cout << "after swap " << a << " " << b << endl;
}

File: programming-languages/exercises/python/swap_two_variables.py
# In python much easier due to tuple packing/unpacking happens internally
"""
Packing
t = 1, 2, 3
This becomes: (1, 2, 3)

Unpacking
a, b, c = (1, 2, 3)
"""
a=int(input("Enter: "))
b=int(input("Enter: "))
print(f"Before swap -> {a,b}")
a,b=b,a
print(f"After swap -> {a,b}")


